cmd/dockerregistry/main.go

```diff
@@ -16,6 +18,18 @@ import (
 )
 
 func main() {
+	// Start the profiling server, if requested.
+	profilingAddr := os.Getenv("REGISTRY_PROFILING_ADDR")
```

Comment:
Hard-coded to loopback with no overrides:

https://github.com/openshift/origin/blob/master/pkg/cmd/server/start/kubernetes/kubernetes.go#L53
https://github.com/openshift/origin/blob/master/pkg/cmd/server/start/start_allinone.go#L257

Did I miss somewhere else? I'm not seeing any way to configure the bind addr.